### PR TITLE
fix(db-postgres): relationship fields: ilike used for contains operations

### LINKED ISSUE #4220: db-postgres: contains operator not working for relationships
### Link to reproduction

https://github.com/payloadcms/payload/pull/4212

### Describe the Bug

`contains` operator does not work for hasMany relationships.

Fixed by this PR - it also includes a failing test: https://github.com/payloadcms/payload/pull/4212

### To Reproduce

-

### Payload Version

-

### Adapters and Plugins

_No response_

### PR DESCRIPTION
**Pull request by AlessioGr**
## Description

Fixes #4220 

- [ ] I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.

## Type of change

- [ ] Chore (non-breaking change which does not add functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Change to the [templates](../templates/) directory (does not affect core functionality)
- [ ] Change to the [examples](../examples/) directory (does not affect core functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/db-postgres/src/queries/parseParams.ts [hunk 1]
```diff
@@ -148,6 +148,10 @@ export async function parseParams({
                   break
                 }
 
+                if (field.type === 'relationship') {
+                  operator = 'equals'
```
Comment (DanRibbens) [inline packages/db-postgres/src/queries/parseParams.ts:152]: We can't just assume they want an `equals` operator. This wouldn't work if you were querying with `exists`, `in`, or `like`.

File: test/relationships/int.spec.ts [hunk 2]
```diff
@@ -274,6 +274,90 @@ describe('Relationships', () => {
...
+          collection: treeSlug,
+          data: {
+            text: 'Tree 4',
+            parent: tree3.id,
+          },
+        })
+
+        const validParents = [tree2.id, tree3.id]
+
+        console.log('VALID PARENTS', validParents)
```
Comment (DanRibbens) [inline test/relationships/int.spec.ts:340]: Delete

**Review by DanRibbens — CHANGES_REQUESTED** [review]
Looks good

**Comment by denolfe** [comment]
@DanRibbens @AlessioGr Should this be revisited?

**Review by DanRibbens — APPROVED** [review]

**Pull request merged**